build.gradle

```diff
@@ -10,9 +10,6 @@
 
 buildscript {
     repositories { 
-        jcenter()
```

Comment:
Is there a reason maven central is removed. (It may get added by default). 
JCenter is going away so that removal is fine